task.go

```diff
@@ -113,6 +113,12 @@ func (task Task) String() string {
 
 // ParseTask parses the input text string into a Task struct.
 func ParseTask(text string) (*Task, error) {
+    // bail out early by returning a nil pointer
```

Comment:
Use tab instead whitespace. Maybe you can run gofmt or something to format this.